pages/grants/grants-and-awards.md

```diff
@@ -93,8 +93,25 @@ The application will request the following details:
   will obtain if you receive the US-RSE grant.
 - How will this proposal, if granted, benefit US-RSE or the wider RSE community?
 - Describe how the proposal aligns with [the mission of US-RSE](https://us-rse.org/about/mission/).
-- Describe your plan for evaluating the success of the proposal. 
+- Describe your plan for evaluating the success of the proposal.
 - As an expectation of funding, we will ask you to write a short post for the
   US-RSE news and/or newsletter describing the proposal and its impact and/or
   outcomes. Are there additional outputs or deliverables you expect to result
   from this proposal?
+
+### FAQs
+
+<details>
+  <summary><b>Q: Can individuals submit multiple proposals?</b></summary>
+    A: Yes, individuals can submit multiple proposals.
+</details>
+
+<details>
+  <summary><b>Q: Can a travel grant be retroactive (i.e., used for a conference that has recently passed from before the review period?)</b></summary>
+    A: No; applications must be for a future event. Applications are reviewed every quarter. See the <a href="https://us-rse.org/grants-and-awards/">website</a> for more details on upcoming deadlines.
```

Comment:
```suggestion
    A: No; applications must be for a future event. Applications are reviewed every quarter. See the above for more details on upcoming deadlines.
```
Since they are already on that page...

Reply:
Good catch. I fixed it with the push below.